mob.go

```diff
@@ -224,7 +220,7 @@ func config(c Configuration) {
 	say("MOB_TIMER" + "=" + c.MobTimer)
 }
 
-func parseArgs(args []string) (command string, parameters []string) {
+func parseArgs(args []string, configuration Configuration) (command string, parameters []string, configuration2 Configuration) {
```

Comment:
not sure if we need the configuration2 here. but I can fix that later

Reply:
I don’t know enough Go to really know a better way...